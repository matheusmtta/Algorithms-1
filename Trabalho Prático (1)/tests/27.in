40 i
45 i
6 i
52 i
40 r
123 p
238 p
37 i
11 i
45 r
37 r
11 r
10 i
14 i
52 r
4 i
4 r
30 i
33 i
9 p
173 p
24 i
145 p
14 r
33 r
146 p
25 i
61 i
30 r
13 i
16 i
98 p
53 i
229 p
42 i
65 i
140 p
19 i
53 r
135 p
3 i
58 i
24 r
16 r
61 r
