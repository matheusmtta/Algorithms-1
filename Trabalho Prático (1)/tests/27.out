5
6
3
6
6
7
3
6
3
3
